pallets/committee/src/lib.rs

```diff
@@ -64,6 +64,9 @@ pub mod pallet {
 		/// Duration (in blocks) of the voting period
 		type VotingPeriod: Get<Self::BlockNumber>;
 
+		/// Minimum number of council members
+		type MinCouncilMembers: Get<usize>;
+
```

Comment:
can we use the MinCouncilVotes instead? this is essentially what limits how many council members are need to remain operational

Reply:
If we follow the docs in polkadotindex.com, there are 5 council members by default, 2 left for open seat, total for 7, and the minCouncilMembers will be specified by 5